TestAssets/TestPackages/dotnet-dependency-context-test/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-preview1-*",
+  "version": "1.0.0-preview2-*",
```

Comment:
Hmm I guess even if we fixed this, we're still going pass preview1 in the versionsuffix. It might be better not to fork that at the moment.